src/util.ts

```diff
@@ -1437,9 +1438,7 @@ export function isValidRGB(rgb: string): boolean {
  * @returns True if the string is valid, false otherwise.
  */
 export function isValidRGBA(rgba: string): boolean {
-  rgba = rgba.replace(' ', '')
-  const isOk = /rgba\((\d{1,3}),(\d{1,3}),(\d{1,3}),(0?.{1,3})\)/i.test(rgba)
-  return isOk
+  return rgbaRE.test(rgba)
```

Comment:
I think you changed the functionality here!
The function `isValidRGBA` was just testing if a expression is a valid RGBA. Now it is also valid if it a plain RGB (short/full). I don't thing we should do this!

Reply:
No it doesn't. There is `'rgb(0, 12, 123)'` among invalid values in the test and it correctly returns false.

Reply:
Ah! I see! the css [rgb()](https://www.w3schools.com/cssref/func_rgb.asp) function is not recognized as valid here at all. I think it is missing in line 15/16.
Do you think we should make this improvement?

Reply:
I don't know what you mean. Lines 15 and 16 are `#000` and `#000000` format colors.

Reply:
it is also possible to format colors like so `rgb( 123, 200, 100 )`. This is still missing here as I see it. Of course this is not you false but maybe we should add it to the big regexp in line 17 ?

Reply:
Yeah, the RGB guard has inlined regexp. I'll probably change it so that they all follow the same pattern.

Reply:
Hmm, we don't really validate colors using these. The only place where we use them (`isValidRGB` and `isValidRGBA`) is the color picker.